packages/generator-one-app-module/generators/app/templates/base-module/src/csp.js

```diff
@@ -0,0 +1,28 @@
+import contentSecurityPolicyBuilder from 'content-security-policy-builder';
+import ip from 'ip';
+
+export default contentSecurityPolicyBuilder({
```

Comment:
We should probably add a comment explaining what this config is doing and why. Perhaps link to the module config docs for CSP in the one app readme?

Reply:
added!